aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -306,45 +306,46 @@ public static ModifyDbInstanceRequest modifyDbInstanceRequest(
-                .enableIAMDatabaseAuthentication(desiredModel.getEnableIAMDatabaseAuthentication())
-                .enablePerformanceInsights(desiredModel.getEnablePerformanceInsights())
-                .licenseModel(desiredModel.getLicenseModel())
-                .masterUserPassword(desiredModel.getMasterUserPassword())
-                .maxAllocatedStorage(desiredModel.getMaxAllocatedStorage())
-                .monitoringInterval(desiredModel.getMonitoringInterval())
-                .monitoringRoleArn(desiredModel.getMonitoringRoleArn())
-                .multiAZ(desiredModel.getMultiAZ())
-                .networkType(desiredModel.getNetworkType())
-                .optionGroupName(desiredModel.getOptionGroupName())
-                .performanceInsightsKMSKeyId(desiredModel.getPerformanceInsightsKMSKeyId())
-                .performanceInsightsRetentionPeriod(desiredModel.getPerformanceInsightsRetentionPeriod())
-                .preferredBackupWindow(desiredModel.getPreferredBackupWindow())
-                .preferredMaintenanceWindow(desiredModel.getPreferredMaintenanceWindow())
-                .promotionTier(desiredModel.getPromotionTier())
-                .storageType(desiredModel.getStorageType())
-                .tdeCredentialArn(desiredModel.getTdeCredentialArn())
-                .tdeCredentialPassword(desiredModel.getTdeCredentialPassword())
+                .autoMinorVersionUpgrade(TranslatorUtils.difference(previousModel.getAutoMinorVersionUpgrade(), desiredModel.getAutoMinorVersionUpgrade()))
+                .backupRetentionPeriod(TranslatorUtils.difference(previousModel.getBackupRetentionPeriod(), desiredModel.getBackupRetentionPeriod()))
+                .caCertificateIdentifier(TranslatorUtils.difference(previousModel.getCACertificateIdentifier(), desiredModel.getCACertificateIdentifier()))
+                .copyTagsToSnapshot(TranslatorUtils.difference(previousModel.getCopyTagsToSnapshot(), desiredModel.getCopyTagsToSnapshot()))
+                .dbInstanceClass(TranslatorUtils.difference(previousModel.getDBInstanceClass(), desiredModel.getDBInstanceClass()))
+                .dbInstanceIdentifier(TranslatorUtils.difference(previousModel.getDBInstanceIdentifier(), desiredModel.getDBInstanceIdentifier()))
+                .dbParameterGroupName(TranslatorUtils.difference(previousModel.getDBParameterGroupName(), desiredModel.getDBParameterGroupName()))
+                .dbPortNumber(translatePortToSdk(TranslatorUtils.difference(previousModel.getPort(), desiredModel.getPort())))
+                .deletionProtection(TranslatorUtils.difference(previousModel.getDeletionProtection(), desiredModel.getDeletionProtection()))
+                .domain(TranslatorUtils.difference(previousModel.getDomain(), desiredModel.getDomain()))
+                .domainIAMRoleName(TranslatorUtils.difference(previousModel.getDomainIAMRoleName(), desiredModel.getDomainIAMRoleName()))
```

Comment:
NIT: this still looks too verbose. I'd propose to change it to `DiffUtils.diff`. The name would rather be shorter: we are not adding any more clarity and only making the code less readable. Let's import this method statically: I believe in this case using a static import is justified for the sake of keeping it all-clear.